### PR TITLE
[EDU-2244] BUILD MENU PT Rules Engine

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Translated Rules Engine guide

### Additional links

[EDU-2243]: https://aziontech.atlassian.net/browse/EDU-2244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[EDU-2244]: https://aziontech.atlassian.net/browse/EDU-2244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: Como criar regras para automatizar comportamentos usando o Rules Engine
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:2]:
```suggestion
title: Como criar regras para automatizar comportamentos com o Rules Engine
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+description: Integre regras em sua aplicação para definir tarefas para cenários específicos sem alterar seu código existente.
+meta_tags: build
+namespace: docs_guides_build_work_with_rules_engine
+permalink: /documentation/products/guides/build/work-with-rules-engine/
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:6]: ajustar link

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+menu_namespace: buildMenu
+---
+
+Implementar regras em sua aplicação usando o [Rules Engine](/en/documentation/products/edge-application/rules-engine/) permite que você determine as tarefas que ela deve executar em cenários específicos sem modificar o código-fonte da sua aplicação.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:10]:
```suggestion
Implementar regras em sua aplicação usando o [Rules Engine](/pt-br/documentacao/produtos/edge-application/rules-engine/) permite que você determine as tarefas que ela deve executar em cenários específicos sem modificar o código-fonte da sua aplicação.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+:::caution[warning]
+Para desbloquear todas as funcionalidades do Rules Engine, você deve ativar o módulo [Application Acceleration](/en/documentation/products/edge-application/application-acceleration/). Se o **Application Acceleration** estiver ativado, a transferência de dados pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/en/pricing/) para mais informações.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:13]:
```suggestion
Para desbloquear todas as funcionalidades do Rules Engine, você deve ativar o módulo [Application Acceleration](/pt-br/documentacao/produtos/edge-application/application-acceleration/). Se o **Application Acceleration** estiver ativado, a transferência de dados pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para mais informações.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+Este guia demonstra o processo de criação de regras com um exemplo prático, mas a maioria das regras será única para cada aplicação e suas necessidades. Primeiro, para ativar o módulo **Application Acceleration**:
+
+1. [Acesse o RTM](https://manager.azion.com).
+2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Edge Application**.
+3. Clique na edge application que você deseja configurar.
+4. Ative o módulo **Application Acceleration**.
+5. Clique no botão **Save**.
+
+Além de ativar variáveis como [device groups](/en/documentation/products/guides/build/create-device-groups/), [configurações de cache](/en/documentation/products/edge-application/cache-settings/) e [origens](/en/documentation/products/guides/build/work-with-origins/), você pode criar uma grande variedade de regras nas fases de solicitação e resposta.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:26]: lembrar ajustar links PT

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+5. Na seção **Criteria**, selecione a variável `${uri}`.
+6. Como operador de comparação, selecione **começa com**.
+7. Como argumento, adicione o valor `/`.
+8. Na seção **Behaviors**, selecione **Filter Response Header**.
+9. Como argumento, adicione `Server`.
+10. Clique no botão **Save**.
+
+## Via API
+
+1. Execute a seguinte solicitação `PATCH` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/en/documentation/products/guides/personal-tokens/) e a variável `<edge_application_id>` por [seu ID de edge application(/en/documentation/products/guides/build/configure-main-settings/) para ativar o módulo [Application Acceleration](/en/documentation/products/edge-application/application-acceleration/):
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:47]:
```suggestion
1. Execute a seguinte solicitação `PATCH` em seu terminal, substituindo `[TOKEN VALUE]` por seu [personal token](/pt-br/documentacao/produtos/gestao-de-contas/personal-tokens/) e a variável `<edge_application_id>` por [seu ID de edge application(/en/documentation/products/guides/build/configure-main-settings/) para ativar o módulo [Application Acceleration](/pt-br/documentacao/produtos/edge-application/application-acceleration/):
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+        "order": 1,
+        "description": null
+    }
+}
+```
+
+5. Aguarde alguns minutos para que as alterações se propaguem.
+
+:::tip
+Consulte a [documentação da API Azion](https://api.azion.com/) e a [especificação OpenAPI](https://github.com/aziontech/azionapi-openapi/) para saber mais sobre todos os recursos disponíveis via API.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:122]:
```suggestion
Consulte a [documentação da API da Azion](https://api.azion.com/) e a [especificação OpenAPI](https://github.com/aziontech/azionapi-openapi/) para saber mais sobre todos os recursos disponíveis via API.
```

File: src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx [hunk 8]
```diff
@@ -0,0 +1,123 @@
+---
+title: Como criar regras para automatizar comportamentos com o Rules Engine
+description: Integre regras em sua aplicação para definir tarefas para cenários específicos sem alterar seu código existente.
+meta_tags: build
+namespace: docs_guides_build_work_with_rules_engine
+permalink: /documentation/products/guides/build/work-with-rules-engine/
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/build-journey/edit-edge-app/edit-rules-engine/rules-engine.mdx:6]:
```suggestion
permalink: /documentacao/produtos/guias/build/trabalhe-com-rules-engine/
```

**Pull request closed without merging**